agent.go

```diff
@@ -33,6 +33,7 @@ type Agent struct {
 	Authorized      bool        `json:"authorized,omitempty"`
 	Env             AgentEnv    `json:"environment,omitempty"`
 	Tags            interface{} `json:"tags,omitempty"`
+	Modules         []string    `json:"modules,omitempty"`
```

Comment:
I'm wondering if we should include this information in the agent's environment element here (Env), vs. putting it in a new part of the heartbeat. The advantage of putting it in the environment is we get this information inserted into the database without any changes there. If we want to put it outside the environment, I think we should modify the scheduler code to also insert this information into the agents table.

See for example here: https://github.com/mozilla/mig/blob/master/mig-scheduler/agents_queues.go#L125 where we process the incoming heartbeat, and here: https://github.com/mozilla/mig/blob/master/database/agents.go#L97

One of the goals with this is to be able to target agents running specific modules via -t to the command line and so on, so we will need this information in the database.